### PR TITLE
Added new header file and changed updated current headers

### LINKED ISSUE #4: Update header files

### PR DESCRIPTION
**Pull request by paulahasstenteufel**
This resolves #4

### CODE REVIEW

File: SwiftGame.xcworkspace/xcshareddata/IDETemplateMacros.plist [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	<key>FILEHEADER </key>
+	<string>//
+// ___PACKAGENAME___
+// Created by: ___FULLUSERNAME___ and Pedro Ferrari on ___DATE___
+// Copyright (c) 2018 ThePugas
+//
+//  _________________________________________
+// |   _____ _       _____                   |
+// |  |_   _| |_ ___|  _  |_ _ ___ ___ ___   |
+// |    | | |   | -_|   __| | | . | .&apos;|_ -|  |
```
Comment (thepedroferrari) [inline SwiftGame.xcworkspace/xcshareddata/IDETemplateMacros.plist]: Check if &apos; is a safe character or it does produces errors when automating the header creation on new files.

**Review by thepedroferrari — COMMENTED** [review]
Verify commented lines

File: SwiftGame.xcworkspace/xcshareddata/IDETemplateMacros.plist (hunk 1 above)
Comment (paulahasstenteufel) [reply SwiftGame.xcworkspace/xcshareddata/IDETemplateMacros.plist]: New headers are being created correctly

**Comment by paulahasstenteufel** [comment]
Updated file location

**Review by thepedroferrari — APPROVED** [review]

**Pull request merged**